src/main/cpp/t05_palindrom.cpp

```diff
@@ -23,5 +23,17 @@
 using namespace std;
 
 int t05_palindrom() {
-
+    string s;
+    cin>>s;
+    int l = s.length();
+    int i=0,j=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'j' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513132)